### PR TITLE
Translate used ngb components

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Fix build warnings and translate alt text of pagination component.

## Context

#402, #404

## Changes

* fix target local on all translation files
* add used ngb components (pagination progress bar) strings to translation files
* provide translation strings for all symbols

## How to test this PR

\-

### CODE REVIEW

**Comment by blacksponge** [comment]
Accessible Rich Internet Applications, more information: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

**Comment by blacksponge** [comment]
For context, in the pagination module those string are used in `aria-label` attributes.

**Review by matsduf — APPROVED** [review]

File: src/locale/messages.da.xlf [hunk 1]
```diff
@@ -323,6 +323,42 @@
...
+      </trans-unit>
+      <trans-unit id="ngb.pagination.next-aria" datatype="html">
+        <source>Next</source>
+      </trans-unit>
+      <trans-unit id="ngb.pagination.last-aria" datatype="html">
+        <source>Last</source>
+      </trans-unit>
+      <trans-unit id="ngb.progressbar.value" datatype="html">
+        <source>
+          <x id="INTERPOLATION" equiv-text="rogress bar."/>
```
Comment (pnax) [inline src/locale/messages.da.xlf:356]: `equiv-text="rogress bar."` : is there a typo `rogress-bar` -> `progress-bar`?

**Review by pnax — COMMENTED** [review]
### v2023.1 release testing

* while reviewing I found something that looks like a typo (not problematic).
* while building the GUI I got an warning message that might come from an update in a later PR (#420), see https://github.com/zonemaster/zonemaster-gui/pull/420/#discussion_r1211759280

**Pull request merged**